cartographer_ros/cartographer_ros/node.cc

```diff
@@ -261,6 +261,16 @@ void Node::PublishLocalTrajectoryData(const ::ros::TimerEvent& timer_event) {
         node_options_.use_pose_extrapolator
             ? ToRos(now)
             : ToRos(trajectory_data.local_slam_data->time);
+
+    // Suppress publishing if we already published a transform at this time.
+    // Due to 2020-07 changes to geometry2, tf buffer will issue warnings for
+    // repeated transforms with the same timestamp
+    if (last_published_tf_stamp_.find(entry.first) !=
+            last_published_tf_stamp_.end() &&
```

Comment:
If you want to check if a simple `std::map` has the key, you can simplify this to 
```
if (last_published_tf_stamps.count(entry.first) && ...
```
(`last_published_tf_stamps.count(entry.first) != 0` would also be possible, but the count gets casted to false if it's zero.)